cli/js/40_jupyter.js

```diff
@@ -412,6 +419,25 @@ function enableJupyter() {
     return;
   }
 
+  // Override confirm and prompt because they depend on a tty
+  // and in the Deno.jupyter environment that doesn't exist.
+  async function confirm(message = "Confirm") {
+    const answer = await input(`${message} [y/N] `, false)
+    return answer === "Y" || answer === "y";
+  }
+
+  async function prompt(message = "Prompt", defaultValue = "", { password = false } = {}) {
+    const answer = await input(`${message} [${defaultValue}] `, password)
+
+    if(answer === "") {
+      return defaultValue;
+    }
+
+    return answer;
+  }
+
+  globalThis.confirm = confirm;
+  globalThis.prompt = prompt;
```

Comment:
I'll leave it up to the core Deno folks to determine if this is how they want this global overriden when in a Jupyter context.

Reply:
This is correct 👍